impl.go

```diff
@@ -240,7 +241,7 @@ type Param struct {
 	Type string
 }
 
-func (p Pkg) funcsig(f *ast.Field, cmap ast.CommentMap) Func {
+func (p Pkg) funcsig(f *ast.Field, cmap ast.CommentMap, noComments bool) Func {
```

Comment:
Bool arguments are easy to confuse. Please give this a named type and designated constant values, e.g.

```go
// Docs here
type EmitComments bool

const (
  WithComments EmitComments = true
  WithoutComments EmitComments = false
)
```